website/src/css/pages/_jobs.scss

```diff
@@ -246,27 +246,19 @@
     }
 
     .reviews-carousel {
-        padding: 0 0 100px 0;
         background-color: #fff;
```

Comment:
Can swap this out for `bg-white` as well — [Bootstrap default](https://getbootstrap.com/docs/4.0/utilities/colors/#background-color)!

Reply:
Whoops, I did before but didn't commit it I guess. It's in now 👍